### PR TITLE
Beta tester comments

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 1]
```diff
@@ -34,7 +34,7 @@ Yet haven't you also told me lies?"
 -- 「む。なにその言い方。すっごくいやらしいんですけど」
 // Skihi VA is somewhat accusing, sulky so 言い方 -> tone
 "Hmph. What's with that tone?
- You're being extremely unpleasant you know."
+ You're being extremely unpleasant, you know."
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3F.txt:37]: kinda think revert this one. Grammatically correct or no, I think it reads cleaner without the comma

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt [hunk 2]
```diff
@@ -857,7 +857,7 @@ Once I've cut off his supply of blood,
 -- Page 395, Offset 17559.
 -- 弱っているのを隠したところで、そんな体じゃ逆に殺されるようなものなんだろ……！　なら―――」
 Even if you hide the fact that you're weak right now,
- with your body in that state you're just going to get yourself killed...!
+ with your body in that state, you're just going to get yourself killed...!
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3F.txt:860]: not sure this one makes much sense either but I'm actually not confident in that judgement

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt [hunk 3]
```diff
@@ -601,8 +601,8 @@ You shouldn't strain yourself to make out things you can't see.
 -- こんな希少な魔眼持ちなのに平和主義者とか、おいしい獲物にも程があるし！　よく今まで生きてこられたわね、ひょいってつまんで食べられるわよ、普通！」
 To think that you're a pacifist despite being in possession of such rare Mystic Eyes!
  Do you have any idea how juicy of a target this makes you?
- You've honestly done well to make it this far,
- normally someone would've snatched you up and gobbled you down by now!"
+ You've honestly done well to make it this far.
+ Normally, someone would've snatched you up and gobbled you down by now!"
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:605]: isn't the expression "gobble you up"?

File: script/Arcueid/Day 7/07_00_ARC07_5.txt [hunk 4]
```diff
@@ -22,7 +22,7 @@ I let out a sigh of gratitude.
 [sha:1b0d57ed0f16ebe9119ea69afc47f00d9445e392]{
 -- Page 657, Offset 18236.
 -- 秋葉や琥珀さんたちを起こさないように、抜き足で屋敷の中へと入っていった。
-I enter the mansion, treading gingerly so as not to wake Ahika or Kohaku-san.
+I enter the mansion, treading gingerly so as not to wake Akiha or Kohaku-san.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_5.txt:25]: Akiba :sacchincopium:

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 7/07_00_ARC07_3F.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3F.txt:37]: Interesting, I was actually pro- this one. Comma placement is heavily subjective though :aokoheh:

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3G.txt:605]: Yeah, I think I had made it `down` here to make an arc out of snatch **up** / gobble **down**. Can change it to just 
> would've snatched you and gobbled you up by now!

or
> would've plucked you out and gobbled you up by now!

if that's better

**Pull request merged**